home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/CooperationServiceImpl.java

```diff
@@ -86,6 +96,10 @@ public CooperationDto getCooperationById(Long id) {
             .orElseThrow(() -> new NotFoundHomeException(String.format(NOT_FOUND_COOPERATION_FORMAT, id)));
         List<House> houseList = houseRepository.findHousesByCooperationId(toGet.getId());
         toGet.setHouses(houseList);
+        List<Contact> contactList = new ArrayList<>();
+        contactList.addAll(contactRepository.findAllByCooperationIdAndType(toGet.getId(), ContactType.EMAIL));
```

Comment:
Why do we need to load contacts separately by type? Can we load all of them together?

Reply:
At that moment I used functionality what I have